layouts/default.vue

```diff
@@ -13,25 +19,34 @@
         <mtl-navbar-top />
         <nuxt />
       </div>
-    </div>
-
-    <mtl-footer />
+    </div>-->
+    <!-- <mtl-footer /> -->
   </div>
 </template>
 
 <script>
+import { MtlNavMobile, MtlSideBarMobile } from '@/components/index'
 export default {
   components: {
-    MtlNavbarTop: () => import('~/components/navbar/navbar-top'),
-    MtlNavbarLeft: () => import('~/components/navbar/navbar-left'),
-    MtlFooter: () => import('~/components/footer')
+    MtlSideBarMobile,
+    MtlNavMobile
   },
+  data: () => ({
+    currentRoute: ''
+  }),
   computed: {
     openNav() {
       return this.$store.state.navMobile
     }
   },
+  watch: {
+    '$route.query': {
+      handler: 'parseRoute',
+      immediate: true
+    }
+  },
   mounted() {
+    this.parseRoute()
```

Comment:
You don't need this line anymore with `immediate: true`
```suggestion
```

Reply:
yep forgot to delete them thx